src/Reports/Gitblame.php

```diff
@@ -28,7 +28,7 @@ class Gitblame extends VersionControl
      *
      * @param string $line Line to parse.
      *
-     * @return mixed string or false if impossible to recover.
+     * @return array|false|string|string[]|null string or false if impossible to recover.
```

Comment:
```suggestion
     * @return string|false string or false if impossible to recover.
```

What are the other types based on ? I can't find a code path which would lead to those....